### ISSUE TITLE
Strange colouring outside of legend bounds

### ISSUE THREAD

**Issue by elangham** [TerriaJS/aremi-natmap#329]
Both NOM prototype and public versions have unusual display when values exceed the legend extent (goes from red back to green temporarily). See Canowindra Peak day map at 6pm: https://nationalmap.gov.au/renewables/#share=s-blmmmJeeFvvyDDV6. 

Definitely not high priority but perhaps let us know if anything we need to do at ISF end to address

**Comment by meh9** [comment]
Dropping in a screenshot showing the problem.

@elangham can you confirm this is the problem you meant?

<img width="1440" alt="screen shot 2019-02-20 at 22 49 13" src="https://user-images.githubusercontent.com/5283003/53089865-d7322b80-3561-11e9-86ec-3be297bd0d9e.png">

**Comment by elangham** [comment]
Yep that's it. A quick look elsewhere the same thing happens At Maryborough, VIC - appears to be exclusively when you go over the legend bounds
![screen shot 2019-02-20 at 12 22 17 pm copy](https://user-images.githubusercontent.com/33736687/53091591-40865000-350a-11e9-8982-49f1026dc405.jpg)
